.github/workflows/build-test-release.yml

```diff
@@ -13,8 +13,54 @@ on:
                 type: string
                 required: true
 
+env:
+    NODE_VERSION: "16.x"
+
 jobs:
-  dummy:
+    build:
```

Comment:
This doesn't only build, maybe it should be called `build-and-make-release` or something :shrug: 
(or should be split into multiple jobs)

Reply:
done